### PR TITLE
ChimeraStation, better looks in some places, and small fixes

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request

makes some hallway looks better, remove some ugly stuff and ands things in main

## Why It's Good For The Game

Ugly stuff: no
prettly stuff: yes

## Changelog
:cl:
tweak: i really dont know what to put here
/:cl:

### CODE REVIEW

File: _maps/map_files/ChimeraStation/ChimeraStation.dmm [hunk 1]
```diff
@@ -33159,7 +33184,12 @@
 /obj/structure/window/reinforced{
 	dir = 1
 	},
-/mob/living/simple_animal/hostile/retaliate/terry,
+/obj/structure/bed/dogbed/mcgriff,
```
Comment (Kitsunemitsu) [inline _maps/map_files/ChimeraStation/ChimeraStation.dmm:33187]: Very funny, sadly for you I read code now

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Remove Mcgriff please

File: _maps/map_files/ChimeraStation/ChimeraStation.dmm (hunk 1 above)
Comment (DaddyIssues98) [reply _maps/map_files/ChimeraStation/ChimeraStation.dmm:33187]: is not mcgriff, is called mcgrieff, diferrent dog

Comment (Kitsunemitsu) [reply _maps/map_files/ChimeraStation/ChimeraStation.dmm]: Changes still requested,

**Pull request merged**